org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/uriresolver/CacheResourcesManager.java

```diff
@@ -51,7 +52,7 @@
 public class CacheResourcesManager {
 
 	private static final String USER_AGENT_KEY = "User-Agent";
-	private static final String USER_AGENT_VALUE = "LemMinX";
+	private static final String USER_AGENT_VALUE = "LemMinX/"+Platform.getVersion().getVersionNumber() + " ("+Platform.getOS().getName() + " "+ Platform.getOS().getVersion()+")";

```

Comment:
I fear that we can have the same problem with Java/1.8 user agent which contains à.

Reply:
No it's fine. I just tried: 

```java
	public static void main(String[] args) throws Exception {
		String actualURI = "https://lime.software/xsd/project-1.0.0.xsd";
		URL url = new URL(actualURI);
		URLConnection conn = url.openConnection();
		//conn.addRequestProperty("User-Agent", "Java/1.8"); // throws a 403
		conn.addRequestProperty("User-Agent", "LemMinX/1.8"); // works fine
		conn.getInputStream();
	}
```

Reply:
What? How it is possible? Thanks for the info.